docs/usage.md

```diff
@@ -36,11 +36,14 @@ Flush varnish:
 
 Connect to redis:
 
-    warden env exec redis redis-cli
+    warden redis
 
 Flush redis completely:
 
-    warden env exec -T redis redis-cli flushall
+    warden redis flushall
+
+Run redis continous stat mode
+    warden redis stat
```

Comment:
Is it still useful to have this shortcut if `warden redis --stat` functions? Open to either, but we should update `WARDEN_CMD_ANYARGS` (see here: https://github.com/davidalger/warden/blob/develop/bin/warden#L41) to the following so that Warden will pass along the args having a leading double hyphen instead of failing with an unkown arg error:

```
declare WARDEN_CMD_ANYARGS=(svc env db redis sync shell debug)
```

Reply:
I think it's better to use `warden redis --stat` here. What do you think?